### DISCUSSION TITLE
Link to openapi.json from Scalar API Reference

### DISCUSSION THREAD

**Discussion by tmcw** [scalar/scalar#1911] [Q&A]
The default reference UI has a Download button, but it's a button element that prompts a download, I believe based on client-side information. If I supply a spec URL to run the reference page, I'd love to expose the URL of the `openapi.json` file so that API consumers can see/reference/use it as well.

**Comment by hanspagel** [comment]
Great idea! Sometimes, we can’t easily do it, because we only get the raw content and don’t have an URL. I think we should point to the URL, if we have it, though! Wdyt @marclave?

**Reply by hanspagel** [reply]
I’ve created an issue here: https://github.com/scalar/scalar/issues/2003
